public/v2/apps/dolibarr.json

```diff
@@ -0,0 +1,89 @@
+{
+    "captainVersion": "2",
+    "documentation": "https://github.com/caprover/one-click-apps",
```

Comment:
Please link the documentation where the docker compose was taken from so that in the future if we need to change/add any more env variable or volumes, we know where to look.